actionview/lib/action_view/test_case.rb

```diff
@@ -287,6 +287,7 @@ def method_missing(selector, *args)
           super
         end
       end
+      ruby2_keywords :method_missing
```

Comment:
```suggestion
      ruby2_keywords(:method_missing)
```

This is for consistency with all the other usages of `ruby2_keywords` in the code base.